service/src/main/java/com/commercetools/pspadapter/tenant/TenantFactory.java

```diff
@@ -50,8 +48,6 @@
 
 public class TenantFactory {
 
-    private static final Duration DEFAULT_CTP_CLIENT_TIMEOUT = Duration.ofSeconds(10);
-
     private static final int MAX_PARALLEL_REQUESTS = 30;
```

Comment:
I guess you do not need it here anymore?

Reply:
Yes. Moved to `ClientConfigurationUtil`
https://github.com/commercetools/commercetools-payone-integration/blob/fc906c778544331e99064f96db22726f2a08a0a9/service/src/main/java/com/commercetools/util/ClientConfigurationUtil.java#L27